swagger.yaml

```diff
@@ -3339,8 +3443,11 @@ definitions:
         type: string
         description: A comma-separated list of NTP sources (name or IP) going to be added to all the hosts.
         x-nullable: true
-      operators:
-        $ref: '#/definitions/list-operators'
+      custom_operators:
```

Comment:
i don't think custom is a right name. We talked about olm as type, so lets call them olm_operators